app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/template/store-pickup.html

```diff
@@ -0,0 +1,20 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<li
+    id="store-pickup"
+    data-bind="fadeVisible: isVisible()"
```

Comment:
Use visible attribute directly. Do not use data-bind attribute at all. https://devdocs.magento.com/guides/v2.3/ui_comp_guide/concepts/magento-bindings.html?itm_source=devdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=ko